frontend/src/APIClients/DonorAPIClient.ts

```diff
@@ -0,0 +1,76 @@
+import AUTHENTICATED_USER_KEY from "../constants/AuthConstants";
+import { getLocalStorageObjProperty } from "../utils/LocalStorageUtils";
+import baseAPIClient from "./BaseAPIClient";
+
+const BEARER_TOKEN = `Bearer ${getLocalStorageObjProperty(
+  AUTHENTICATED_USER_KEY,
+  "accessToken",
+)}`;
+
+export type DonorResponse = {
```

Comment:
Ideally I can use `UserDonorDTO` from the backend types but was getting import errors